### PR TITLE
Pools fix

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

* Hide the Virualization > Provisioning tab on Salt systems since this is not supported.
* parse the pageSize in the pool list to a number

## GUI diff

One broken tab removed.

- [X] **DONE**

## Documentation
- No documentation needed: that tab was not documented for Salt systems

- [X] **DONE**

## Test coverage
- No tests: super tiny change

- [X] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [X] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by cbosdo** [comment]
@mcalmer I submitted that PR a while ago, but from what you said the other day it seems that the commit hiding the **Virtualization > Provisioning** tab for Salt systems should not be merged, right?

File: java/code/webapp/WEB-INF/nav/system_detail.xml [hunk 1]
```diff
@@ -189,7 +189,7 @@
     <rhn-tab-url>/rhn/manager/systems/details/virtualization/guests/${sid}</rhn-tab-url>
     <rhn-tab name="Guests" url="/rhn/manager/systems/details/virtualization/guests/${sid}"/>
     <rhn-tab name="Storage" url="/rhn/manager/systems/details/virtualization/storage/${sid}" acl="system_has_salt_entitlement()"/>
-    <rhn-tab name="Provisioning" url="/rhn/systems/details/virtualization/ProvisionVirtualizationWizard.do" acl="system_feature(ftr_kickstart) or system_feature(ftr_snapshotting)"/>
+    <rhn-tab name="Provisioning" url="/rhn/systems/details/virtualization/ProvisionVirtualizationWizard.do" acl="system_has_management_entitlement(); system_feature(ftr_kickstart) or system_feature(ftr_snapshotting)"/>
```
Comment (mcalmer) [inline java/code/webapp/WEB-INF/nav/system_detail.xml]: Not sure if this is the best way. I think correct would be to add ftr_kickstart feature to salt management.

**Comment by mcalmer** [comment]
I am a bit lost here. You say you want to "hide" it, but the change seems to "show" it.

Please check what you want to do and if the code really does it.

**Comment by cbosdo** [comment]
@mcalmer I submitted that PR a while ago, but from what you said the other day it seems that the commit hiding the **Virtualization > Provisioning** tab for Salt systems should not be merged, right?

> I am a bit lost here. You say you want to "hide" it, but the change seems to "show" it.
> 
> Please check what you want to do and if the code really does it.

I'm using it daily here. This adds an and condition on the Management Entitlement to ensure it doesn't show up on Salt minions.

File: java/code/webapp/WEB-INF/nav/system_detail.xml (hunk 1 above)
Comment (mcalmer) [reply java/code/webapp/WEB-INF/nav/system_detail.xml]: it was has "management" entitlement. So this is ok

**Pull request merged**